### DISCUSSION TITLE
What is the best way to handle different encoding with DuckDB?

### DISCUSSION THREAD

**Discussion by cfahlgren1** [duckdb/duckdb#8438] [Q&A]
For DuckDB in production, with different file encoding such as `latin-1`, what is the best way to handle this data and still use DuckDB? Convert it to `UTF-8` and then use it?

**Comment by EED85** [comment]
Hi,
you added encoding `latin-1` to the `read_csv` function, i love it.
For consistency, it would be great, if you added the `latin-1` encoding either to the `read_text` function and/or the `decode` function.
